Library/Homebrew/tap.rb

```diff
@@ -112,6 +112,16 @@ def to_hash
     }
   end
 
+  def load_tap_formula_renames
+    begin
+      path.cd do
+        load "tap_formula_renames.rb"
```

Comment:
I suggest another name such as `.tap_formula_renames` so it wouldn't be treated as formula. I know you have excluded it from `Tap#formula_files`, but considering other existing tools out there it would be best to distinguish them in file name at first place.